schema.go

```diff
@@ -51,6 +52,10 @@ const (
 	CryptoSHA256 = "http://www.w3.org/2001/04/xmlenc#sha256"
 )
 
+const (
+	RFC3339NanoModified = "2006-01-02T15:04:05.9999999Z07:00"
```

Comment:
Call this `SAMLTimeFormat` and explain what it is / how it differs from RFC3339 and why